### DISCUSSION TITLE
Bluemap

### DISCUSSION THREAD

**Discussion by vxnsin** [GriefPrevention/GriefPrevention#1849] [Ideas]
Its really cool wen Bluemaps get addet to see Claimed Areas with the Owner and some infomation

**Comment by bigpresh** [comment]
That's something for an addon to do.

[BlueBridge](https://www.spigotmc.org/resources/bluebridge.82954/) claims to do just that, but I have no experience with it and it doesn't appear to have been updated for a little while, so I don't know how well it'll work for you - but worth a try!
